online/src/main/scala/ai/chronon/online/Fetcher.scala

```diff
@@ -190,23 +190,38 @@ class Fetcher(val kvStore: KVStore,
             val baseMap = internalMap ++ externalMap
             // used for derivation based on ts/ds
             val tsDsMap: Map[String, AnyRef] =
-              Map("ts" -> (requestTs).asInstanceOf[AnyRef], "ds" -> (requestDs).asInstanceOf[AnyRef])
-            val derivedMap: Map[String, AnyRef] = Try(
+              Map("ts" -> requestTs.asInstanceOf[AnyRef], "ds" -> requestDs.asInstanceOf[AnyRef])
```

Comment:
Given this change, `ts` and `ds` are now reserved keywords. Can we add some validations in join.py when creating `Derivation` to prevent people from using `ts` and `ds` as names of derived features? Otherwise this will introduce bugs that are hard to debug.

Reply:
Not have to be in this PR

Reply:
Sounds good. Let me add that!